tests/util/server/src/lib.rs

```diff
@@ -674,21 +674,43 @@ pub fn wildcard_match_detailed(
   let parts = parse_wildcard_pattern_text(&pattern).unwrap();
 
   let mut was_last_wildcard = false;
+  let mut was_last_wildline = false;
   for (i, part) in parts.iter().enumerate() {
     match part {
       WildcardPatternPart::Wildcard => {
         output_lines.push("<WILDCARD />".to_string());
       }
+      WildcardPatternPart::Wildline => {
+        output_lines.push("<WILDLINE />".to_string());
+      }
+      WildcardPatternPart::Wildnum(times) => {
+        if current_text.len() < *times {
+          output_lines
+            .push(format!("==== HAD MISSING WILDNUM({}) ====", times));
```

Comment:
Nit: maybe we should make this WILDN or WILDCHARS? It might be confusing with a numeric pattern.

Reply:
Will change.